### PR TITLE
TTH-12: added yes or no buttons to MatchPage

### PR DESCRIPTION
**Pull request by palak-bhargava**
* **Please check if the PR fulfills these requirements**
- [ ] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Added new yes or no buttons to the match page and a stylesheet for these buttons.

* **What is the current behavior?What does the code do before you made the new changes?** 
The buttons are just displayed on the match page they do not have any functionality as of now. Before I added the code, there were no buttons.

* **What is the new behavior?**
There are buttons on the match page now.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No breaking changes.

* **Other information**:

### CODE REVIEW

File: frontend/src/styles/MPbuttons.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.mp-button{
+    display: inline-block;
+    border-radius: 8px;
+    width: 6em;
+    height: 2em;
+    margin: 10px 20px 10px 20px;
+    text-align: center;
+    font-size: x-large;
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
```
Comment (areebakq) [inline frontend/src/styles/MPbuttons.css]: Maybe use the primary font from App.css?

File: frontend/src/styles/MPbuttons.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+.buttonGroup {
+    width: 100%;
+    text-align: center;
+}
+.mp-button{
```
Comment (areebakq) [inline frontend/src/styles/MPbuttons.css]: A small suggestion would be to add hover styling on the buttons

Comment (palak-bhargava) [reply frontend/src/styles/MPbuttons.css]: I added some hover styling!

**Review by areebakq — APPROVED** [review]
Looks great!!

File: frontend/src/styles/MPbuttons.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+#green{
+    background-color: var(--pastelGreen);
+}
+#green:hover{
+    background-color: var(--pastelLightGreen);
+    color: var(--primaryButtonColor);
+}
+#red{
+    background-color: var(--pastelRed);
+}
```
Comment (roseyume) [inline frontend/src/styles/MPbuttons.css:32]: just to keep our css files matching, add a space between css items {}.

**Review by roseyume — CHANGES_REQUESTED** [review]
Looks good.  I think we need to keep our css files looking similar so add new lines between your css style items like in Navbar.css and in Explanation.css.

**Review by Codubee — APPROVED** [review]
LGTM if you just want to address @roseyume concerns.

File: frontend/src/styles/MPbuttons.css (hunk 3 above)
Comment (palak-bhargava) [reply frontend/src/styles/MPbuttons.css:32]: done!

**Review by thomasabigail — APPROVED** [review]

**Pull request merged**